spec/features/customers/orders/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "Viewing one of a customer's order" do
+  context "when a customer has an order" do
+    scenario "displays the detail page for the order" do
+      order_1, order_2, order_3 = create_list(:all_new_order, 3)
```

Comment:
Yes, Hound, we're making sure that order_1 doesn't appear on the page.  Oh, Hound, you dog.